### PR TITLE
Feature/tt2 169 remove unused notify code

### PR DESCRIPTION
**Pull request by gds-topickering**

### CODE REVIEW

File: src/types/notifyApiSecrets.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-// LEGACY FILE - CODE HAS MOVED TO RESULTS_DELIVERY REPO
-export interface NotifyApiSecrets {
-  notifyApiKey: string
-  notifyTemplateId: string
-}
+// export interface NotifyApiSecrets {
```
Comment (gdsandrewmccafferty) [inline src/types/notifyApiSecrets.ts:1]: Just delete this file, think it's still just commented out just now

Comment (gds-topickering) [reply src/types/notifyApiSecrets.ts:1]: Knew I'd miss one, well done for spotting the deliberate mistake...

**Review by gdsandrewmccafferty — CHANGES_REQUESTED** [review]
Have looked through carefully, looks good, found more trace of Notify you can get rid of
`requestNotSentToNotify` in `/src/constants/loggingCopy.ts`

**Review by gdsandrewmccafferty — APPROVED** [review]

**Review by gdsandrewmccafferty — CHANGES_REQUESTED** [review]
Just spotted the SendEmailQueue is still there in the template

**Review by gdsandrewmccafferty — APPROVED** [review]
Right, think we're there now!

**Pull request merged**